apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -26,25 +26,77 @@ var AnimationPickerBody = React.createClass({
+            sourceHeight={378}
+            frameWidth={132}
+            frameHeight={126}
+            frameCount={9}
+            frameRate={15}
+            label="asterisk_circle (9)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/asterisk_explode.png'}
+            sourceWidth={684}
+            sourceHeight={474}
+            frameWidth={171}
+            frameHeight={158}
+            frameCount={11}
+            frameRate={15}
+            label="asterisk_explode (11)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/asterisk_normal.png'}
+            sourceWidth={396}
+            sourceHeight={126}
+            frameWidth={132}
+            frameHeight={126}
+            frameCount={3}
+            frameRate={15}
+            label="asterisk_normal (3)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/asterisk_stretching.png'}
+            sourceWidth={432}
+            sourceHeight={432}
+            frameWidth={144}
+            frameHeight={144}
+            frameCount={8}
+            frameRate={15}
+            label="asterisk_stretching (8)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/box.png'}
+            sourceWidth={198}
+            sourceHeight={118}
+            frameWidth={66}
+            frameHeight={118}
+            frameCount={3}
+            frameRate={15}
+            label="box (3)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/bubbly.png'}
+            sourceWidth={82}
+            sourceHeight={520}
+            frameWidth={82}
+            frameHeight={130}
+            frameCount={4}
+            frameRate={15}
+            label="bubbly (4)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/ghost_standing.png'}
+            sourceWidth={504}
+            sourceHeight={158}
+            frameWidth={72}
+            frameHeight={158}
+            frameCount={7}
+            frameRate={15}
+            label="ghost_standing (7)"/>
         <AnimationPickerListItem
-            label="asterisk_circle (9)" />
-        <AnimationPickerListItem
-            label="asterisk_circle (9)" />
-        <AnimationPickerListItem
-            label="asterisk_circle (9)" />
+            sourceUrl={'/blockly/media/gamelab/library/platform.png'}
+            sourceWidth={200}
+            sourceHeight={219}
+            frameWidth={200}
+            frameHeight={73}
+            frameCount={3}
+            frameRate={15}
+            label="platform (3)"/>
       </ScrollableList>
```

Comment:
These are temporary, but you're right that I might as well express them as an array of objects (that we'll later get from an external source).